src/clr-core/styles/variables/_variables.global.scss

```diff
@@ -44,7 +44,7 @@ $clr_baselineRem_10: convertBaselineToBase20(240); // 240px
 $clr_baselineRem_15: convertBaselineToBase20(360); // 360px
 
 // Pixel-based utilities
-$clr_baselineRem_-1px: -convertBaselineToBase20(1);
+$clr_baselineRem_-1px: -1 *convertBaselineToBase20(1);
```

Comment:
The recommendation is to not do `$clr_baselineRem_-*px` variables. This could conceivably double the number of rem-sizing variables we have in our SASS.

The recommendation is to multiply whatever negative rem-sizing you need by `-1`.

So in your SASS you would put...

```
whatever: -1 * $clr_baselineRem_1px
```